src/internet/seafile/seafileservice.cpp

```diff
@@ -196,6 +217,15 @@ void SeafileService::Connect() {
 }
 
 void SeafileService::UpdateLibraries() {
+  if (indexing_task_id_ == -1) {
+    indexing_task_id_ =
+        app_->task_manager()->StartTask(tr("Build index of Seafile"));
```

Comment:
Are you sure ? Because this task is different of the CloudFileService (MaybeAddFileToDatabase) one. The user would have two tasks named "Indexing Seafile..." : 
 * one when the Seafile service try to build the index (by checking which directory of the server was updated with the local tree)
 * one (which could be executed in parallel) to index songs of the updated directories (MaybeAddFileToDatabase)

Anyway I think the current correct form would be "Building Seafile index..." no ?